cluster/proto/api/request.go

```diff
@@ -76,13 +76,12 @@ type QueryShardOwnerResponse struct {
 	Owner        string
 }
 
-type QueryTenantShardRequest struct {
-	Class  string
-	Tenant string
+type QueryTenantsShardsRequest struct {
+	Class   string
+	Tenants []string
 }
 
-type QueryTenantShardResponse struct {
-	Tenant         string
-	ActivityStatus string
-	SchemaVersion  uint64
+type QueryTenantsShardsResponse struct {
+	Tenants       map[string]string // map[tenant]status
```

Comment:
discussion(non-blocking): would it be clearer to rename the map to `TenantsToActivity` or `TenantsActivity` or `TenantsActivityStatus` ?